convert2rhel/unit_tests/checks_test.py

```diff
@@ -856,14 +856,14 @@ def test_bad_kernel_substring(kernel_release, exp_return, monkeypatch):
     (
         (
             "4.18.0-240.22.1.el8_3.x86_64",
-            "kernel-core",
+            "4.18.0&240.22.1.el8_3&x86_64&kernel-core",
             "05b555b38483c65d",
             "yajl.x86_64",
             False,
         ),
         (
             "4.18.0-240.22.1.el8_3.x86_64",
-            "kernel-core",
+            "4.18.0&240.22.1.el8_3&x86_64&kernel-core",
```

Comment:
Shouldn't we test with multiple kernel packages in different orders to make sure we meet the acceptance criteria?

Reply:
good point! let me work on that

Reply:
So, @SpyTec, I came back to this code and tested it once again, I just validated with the help of the integration test that @kokesak made that we only return 1 package in get_installed_pkg_objects because we are passing the NEVRA of the package in the search. I don't think there is a way to return more than one package to be returned in that function, since we are getting the package NEVRA from a file located in vmlinuz_path = "/boot/vmlinuz-%s" % kernel_release, and, assuming that the user has a custom kernel installed (e.g: an Oracle Linux kernel installed on a CentOS Linux system), and, the file located in vmlinuz_path points to that package, then I'm not sure if we will return anything else.

The test the way it is right now is somewhat correct, as it tries to validate (in)correct package signatures.

Probably I misunderstood your question in the first place, too. Could you provide a simple example here in the comments, so I can try to see if that would be worth it to include?

Reply:
> we only return 1 package in get_installed_pkg_objects because we are passing the NEVRA of the package in the search

Ohhh gotcha, then yeah you can ignore my suggestion